frontend/packages/kubevirt-plugin/src/components/vms/vm.tsx

```diff
@@ -125,27 +116,25 @@ const VMHeader = (t: TFunction) => () =>
     tableColumnClasses,
   );
 
-const PendingChanges: React.FC = () => {
-  const { t } = useTranslation();
-  return <div className="kv-vm-row_status-extra-label">{t('kubevirt-plugin~Pending changes')}</div>;
-};
-
-const VMRow: RowFunction<VMRowObjType> = ({ obj, index, key, style }) => {
-  const { vm, vmi, vmImport } = obj;
-  const { name, namespace, node, creationTimestamp, uid, vmStatusBundle } = obj.metadata;
+const VMRow: RowFunction<VMRowObjType, VmStatusResourcesValue> = ({
+  obj,
+  index,
+  key,
+  style,
+  customData: vmStatusResources,
+}) => {
+  const { vm, vmi } = obj;
+  const { name, namespace, creationTimestamp, uid, node } = obj.metadata;
   const dimensify = dimensifyRow(tableColumnClasses);
 
   let options: KebabOption[];
   let model: K8sKind;
 
-  if (vmImport) {
-    model = VirtualMachineImportModel;
-    options = vmImportMenuActions.map((action) => action(model, vmImport));
-  } else if (vm) {
+  if (vm) {
     model = VirtualMachineModel;
     options = vmMenuActions.map((action) =>
       action(model, vm, {
-        vmStatusBundle,
+        vmStatusBundle: {},
```

Comment:
@rawagner do you think creating a lazy kebab menu is a viable solution here?

Reply:
hm...maybe let the user know you're waiting for status ? Something like disabled item `loading status` etc ?